scripts/convert_lora_weights.py

```diff
@@ -0,0 +1,102 @@
+import sys
+import time
+from pathlib import Path
+from typing import Optional
+
+import lightning as L
+import torch
+import torch.nn as nn
+
+# support running without installing as a package
+wd = Path(__file__).parent.parent.resolve()
+sys.path.append(str(wd))
+
+from lit_llama import LLaMA
+from lit_llama.utils import EmptyInitOnDevice, lazy_load, llama_model_lookup
+from lit_llama.lora import lora
+
+def del_lora_state_dict(model: nn.Module):
+    key_to_delete = []
+    base_model_dict = model.state_dict()
+    for k in base_model_dict:
+        if "lora_" in k:
+            key_to_delete.append(k)
+    for del_key in key_to_delete:
+        del base_model_dict[del_key]
+    del model
```

Comment:
deleting the model inside this function is strange. The `del_lora_state_dict` should just delete the lora state dict, freeing up memory can be done outside